pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -275,6 +278,9 @@ func validateMutatingWebhook(hook *admissionregistration.MutatingWebhook, fldPat
 	if hook.MatchPolicy != nil && !supportedMatchPolicies.Has(string(*hook.MatchPolicy)) {
 		allErrors = append(allErrors, field.NotSupported(fldPath.Child("matchPolicy"), *hook.MatchPolicy, supportedMatchPolicies.List()))
 	}
+	if hook.SideEffects == nil {
+		allErrors = append(allErrors, field.Required(fldPath.Child("sideEffects"), fmt.Sprintf("must specify one of %v", supportedSideEffectClasses.List())))
```

Comment:
unify with create validation error

Reply:
not sure what you mean by this

Reply:
you have `%v` once with a slice and `strings.Join` everywhere else for the possible values.

Reply:
oh, fixed in later commit, will collapse